BinaryEncoding.md

```diff
@@ -390,10 +387,10 @@ out of range, `br_table` branches to the default target.
 
 The `memory_immediate` type is encoded as follows:
 
-| Name | Type | Present? | Description |
-| ---- | ---- | ---- | ---- |
-| flags | `uint8` | always | a bitfield where<br>bit `4` indicates an offset follows<br>bit `7` indicates natural alignment<br>other bits are reserved for future use |
-| offset | `varuint32` | `flags[4] == 1` | the value of the offset |
+| Name | Type | Description |
+| ---- | ---- | ---- |
+| offset | `varuint32` | the value of the offset |
```

Comment:
Just to be clear, with the most recent update to this PR, alignment takes a fixed number of bits (bounded by `log2(memory width)`) and so, when we want to add any ordering flags, they could go after.  LEB128 lets us have an infinite bitfield :)